app/pages/index.vue

```diff
@@ -42,6 +42,7 @@ useSeoMeta({
   <main class="grow w-full lg:w-2/3 mx-auto mt-20 p-2">
     <SiteHeader />
     <h3 class="text-lg leading-snug text-zinc-600 font-bold mt-8 mb-2">Open Positions</h3>
+    <span class="text-sm text-zinc-500">({{ totalPositions }})</span>
```

Comment:
```suggestion
    <span class="text-sm text-zinc-500">{{ totalPositions }}</span>
```

@karthik8239 please don't use round brackets to show no. of job position instead use css border properties.

cc: @amandesai01

Reply:
Done generated  "()" using css